pom.xml

```diff
@@ -6,7 +6,7 @@
     <artifactId>pcc</artifactId>
     <packaging>pom</packaging>
     <name>pcc</name>
-    <version>${revision}</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
In my view, we should use the `revision` as placeholder for the version in our pom files. Please check [this](https://maven.apache.org/maven-ci-friendly.html).

Reply:
Updated. Is this ok?